### DISCUSSION TITLE
1.0.4 release

### DISCUSSION THREAD

**Discussion by bodgit** [hashicorp/packer-plugin-vsphere#645] [Community Help]
I've just discovered I'm bumping into the bug that was reported in #98 and fixed in #159. Is it possible to get a new release that includes this as it was merged post 1.0.3. Currently, using the `vsphere-template` post-processor is broken because of it.

**Comment by tenthirtyam** [comment]
[v1.0.4](https://github.com/hashicorp/packer-plugin-vsphere/releases/tag/v1.0.4) is released.

Ready to close.

cc @nywilken @azr
